src/osemgrep/language_server/Diagnostics.ml

```diff
@@ -21,6 +21,12 @@ let diagnostic_of_match (m : Out.cli_match) =
         |> Yojson.Safe.Util.to_string_option
     | _ -> None
   in
+  let message =
+    match shortlink with
+    | Some s when is_intellij ->
```

Comment:
maybe add a comment here saying why intellij is lame and so we need to do that because they don't respect
the LSP spec where this kind of link should instead be in the XXX field.

Reply:
will do